oak/server/oak_runtime.h

```diff
@@ -38,31 +40,32 @@ namespace oak {
 
 class OakRuntime {
  public:
-  OakRuntime(){};
-
+  OakRuntime() = default;
   virtual ~OakRuntime() = default;
 
   // Initializes a gRPC server. If the server is already initialized, does nothing.
   asylo::Status Initialize(const ApplicationConfiguration& config);
-  asylo::Status StartCompletionQueue(std::unique_ptr<grpc::AsyncGenericService> service,
-                                     std::unique_ptr<grpc::ServerCompletionQueue> queue);
-  grpc::Service* GetGrpcService();
+  asylo::Status Start();
+  asylo::Status Finalize();
```

Comment:
We have Stop, Shutdown and Finalize, it would be good to pick one; personally I vote for ShutDown -- AFAICT it's two words -- but I don't mind which one we choose.

Reply:
I've got `Start()` and `Stop()` [in](https://github.com/daviddrysdale/oak/commit/5c0fc27d684b1ba6285508b118d14b411925be5f) my [pending branch](https://github.com/daviddrysdale/oak/commits/storage++) so I vote for that.

Reply:
Asylo uses Start and Finalize :)

Reply:
And Grpc has Shutdown.

Reply:
As such I am going for Stop.